### PR TITLE
Added SKAN support, bumped wrapper version to 1.1.0

### PR DESCRIPTION
**Pull request by eyalsingular**

### CODE REVIEW

File: mParticle-Singular/MPKitSingular.m [hunk 1]
```diff
@@ -20,12 +20,16 @@ @implementation MPKitSingular
 
 // Wrapper Consts
 #define MPARTICLE_WRAPPER_NAME @"mParticle"
-#define MPARTICLE_WRAPPER_VERSION @"1.0.1"
+#define MPARTICLE_WRAPPER_VERSION @"1.1.0"
```
Comment (e-avidan) [inline mParticle-Singular/MPKitSingular.m:23]: why not 2?

File: mParticle-Singular/MPKitSingular.m [hunk 2]
```diff
@@ -346,13 +352,31 @@ - (nonnull MPKitExecStatus *) execSuccess{
...
+    config.waitForTrackingAuthorizationWithTimeoutInterval = waitForTrackingAuthorizationWithTimeoutInterval;
+    
+    return config;
+}
+
++(void)setSKANOptions:(BOOL)skanEnabled andMode:(BOOL)manualMode withWaitForTrackingAuthorizationWithTimeoutInterval:(NSNumber*)waitTrackingAuthorizationWithTimeoutInterval withConversionValueUpdatedHandler:(void(^_Nullable)(NSInteger))conversionValueUpdatedHandler {
+    isSKANEnabled = skanEnabled;
+    isManualMode = manualMode;
+    conversionValueUpdatedCallback = conversionValueUpdatedHandler;
+    waitForTrackingAuthorizationWithTimeoutInterval = waitTrackingAuthorizationWithTimeoutInterval ? [waitTrackingAuthorizationWithTimeoutInterval intValue] : 0;
```
Comment (e-avidan) [inline mParticle-Singular/MPKitSingular.m:379]: what about negative values?

File: mParticle-Singular/MPKitSingular.m [hunk 3]
```diff
@@ -346,13 +352,31 @@ - (nonnull MPKitExecStatus *) execSuccess{
...
+    
+    config.skAdNetworkEnabled = isSKANEnabled;
+    config.manualSkanConversionManagement = isManualMode;
+    config.conversionValueUpdatedCallback = conversionValueUpdatedCallback;
+    config.waitForTrackingAuthorizationWithTimeoutInterval = waitForTrackingAuthorizationWithTimeoutInterval;
+    
+    return config;
+}
+
++(void)setSKANOptions:(BOOL)skanEnabled andMode:(BOOL)manualMode withWaitForTrackingAuthorizationWithTimeoutInterval:(NSNumber*)waitTrackingAuthorizationWithTimeoutInterval withConversionValueUpdatedHandler:(void(^_Nullable)(NSInteger))conversionValueUpdatedHandler {
```
Comment (e-avidan) [inline mParticle-Singular/MPKitSingular.m:375]: what if this method is called post config? then we don't care?

**Review by e-avidan — APPROVED** [review]

File: mParticle-Singular/MPKitSingular.m (hunk 1 above)
Comment (eyalsingular) [reply mParticle-Singular/MPKitSingular.m:23]: i can change to 2 but it's not a huge change and for internal use anyway

File: mParticle-Singular/MPKitSingular.m (hunk 2 above)
Comment (eyalsingular) [reply mParticle-Singular/MPKitSingular.m:379]: wdym?

File: mParticle-Singular/MPKitSingular.m (hunk 3 above)
Comment (eyalsingular) [reply mParticle-Singular/MPKitSingular.m:375]: This method needs to be called once before mParticle init.
If it's called afterwards we have nothing to do.

**Pull request merged**